### PR TITLE
Update wording in README

### PR DESCRIPTION
**Pull request by OfficialKris**
This PR brings some changes to help prepare the README.md file for a V1.0 release.

Summary of changes (sorted from most significant to least):
- Reduced wording in compiling section which should be reserved for each of the listed links
- Removed `Cygwin` target in README as there is no valid documentation in the link
- Added break for sponsor section
- Added note section for FPA grant notice

Link to forum discussion (pre-reverts): https://forum.freecad.org/viewtopic.php?t=89597

### CODE REVIEW

**Comment by yorikvanhavre** [comment]
Although I'd be OK with some of the rewording, I generally disagree with this PR, The FreeCAD motto was something elaborated by the community for a long time, it cannot be changed arbitrarily. Also disagree with the "FreeCAD has an industry first open data format." which could be seen as untrue as it's not the first open format of the industry. Also, I don't think the README is the place for things like "unlike proprietary apps". It's mostly a technical document and should say so IMHO.

In short, I think such a deep change to the README should be discussed with the community first.

**Comment by luzpaz** [comment]
Indeed, I'm in agreement with Yorik here. The effort is appreciated nonetheless.

**Comment by chennes** [comment]
Flipping this to draft pending further discussion.

**Comment by OfficialKris** [comment]
Pinging since this was updated. 

I removed changes to moto, other references, and reworked untrue wording. I hope the more general changes could be considered as I think this will help potential users consider FreeCAD after the recent changes for V1.0.

**Comment by yorikvanhavre** [comment]
I generally agree with the present form of this PR, but still, IMHO it needs larger community involvement and approval. I would suggest posting a link to this PR on the forum

**Comment by OfficialKris** [comment]
I have reverted the wide ranging commits. I hope the remaining commits could be considered for posterity. For instance, the link to Cygwin is invalid and could lead to confusion for new developers after a V1.0 release.

**Comment by yorikvanhavre** [comment]
backported

**Comment by yorikvanhavre** [comment]
Ok thanks!
